src/app/tools/point_shapes.h

```diff
@@ -103,6 +103,7 @@ class FloodFillPointShape : public PointShape {
       get_pixel(srcImage, pt.x, pt.y),
       loop->getTolerance(),
       loop->getContiguous(),
+      loop->isPixelConnectivityFourConnected(),
```

Comment:
I'm not sure, but is not `floodfill()` receiving true when it's eight-connected instead of four-connected?

Reply:
You're absolutely right -- inconsistent naming there on my part.  Changed the function reflect `floodfill()` receiving `true` when it's eight-connected.